openapi-annotation-processor/src/test/kotlin/io/javalin/openapi/processor/UserCasesTest.kt

```diff
@@ -97,5 +97,19 @@ internal class UserCasesTest : OpenApiAnnotationProcessorSpecification() {
             .isEqualTo("getApiPandaList")
     }
 
+    @OpenApi(
+        path = "/api/panda/{pandaId}/name/<startsWith>",
+        operationId = OpenApiOperation.AUTO_GENERATE,
+        versions = ["should_generate_operation_id_from_path_with_parameters"]
+    )
+    @Test
+    fun should_generate_operation_id_from_path_with_parameters() = withOpenApi("should_generate_operation_id_from_path_with_parameters"){
+        println(it)
+
+        assertThatJson(it)
+            .inPath("$.paths['/api/panda/{pandaId}/name/\u003cstartsWith\u003e'].get.operationId")
```

Comment:
`<>` are some kind of special operators in `inPath`, right?

Reply:
Yeah, I had to carefully flick through the [documentation](https://javalin.io/documentation#before-handlers) again to find why a) we used them in our project and b) what their meaning was.
The `<>` basically enable having `/` in path params as part of the parameter, as far as I understood. I guess this is to be handled differently as well.

Reply:
I meant why it's `/name/\u003cstartsWith\u003e` instead of `/name/<startsWith>` and if it's related some kind of special behavior of [JsonUnit](https://github.com/lukas-krecan/JsonUnit)`#inPath` matcher 😅

Reply:
I see, no not in particular. I wasn't sure on how the JSON would look like and thus used the previous `println` statement in order to get the correct `inPath` argument.
For readability, I have removed the unicode literals 😄